airbyte-ci/connectors/base_images/base_images/python/bases.py

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+from __future__ import annotations
+
+from typing import Final, Set
+
+import dagger
+from base_images import common, sanity_checks
+from connector_ops.utils import ConnectorLanguage  # type: ignore
+
+from .base_bases import PYTHON_3_9_18
+
+
+class AirbytePythonConnectorBaseImage(common.AirbyteConnectorBaseImage):
+
+    compatible_languages = (ConnectorLanguage.PYTHON, ConnectorLanguage.LOW_CODE)
+    base_base_image: Final[common.PublishedImage] = PYTHON_3_9_18
+    image_name: Final[str] = "airbyte/python-connector-base"
+
+    pip_cache_name: Final[str] = "pip-cache"
+    expected_env_vars: Set[str] = {
```

Comment:
💅 super nit: should this be inside the sanity check function since its not used elsewhere?